data/remnant/remnant missions.txt

```diff
@@ -2208,3 +2208,655 @@ mission "Remnant: Face to Maw 2B"
+				accept
+	npc save accompany
+		government "Remnant"
+		personality pacifist coward launching escort
+		fleet
+			names "remnant"
+			variant
+				"Pelican (Heavy Lift)" 6
+	npc
+		government "Remnant"
+		personality launching escort disables plunders
+		fleet "Large Remnant" 3
+	npc evade
+		government "Korath"
+		personality heroic disables plunders harvests entering staying
+		system "Nenia"
+		fleet
+			names "korath"
+			variant
+				"Korath World-Ship B (Ember)"
+				"Korath Raider (Ember)" 3
+	on enter "Nenia"
+		dialog `As you enter the system, you quickly search for the Archon, but it is nowhere to be seen. A Korath hunting party, however, is clearly waiting for you and moves to attack. As their ships converge on you, the pelicans immediately set to work collecting void sprites. As the last one closes its hold again, a confirmation codes flash across your tactical screen. "Good to go."`
+	on enter "Pantica"
+		dialog `You watch vigilantly as the Pelicans with the void sprites dip low into Esquilion's atmosphere and open their bay doors. The void sprites emerge tentatively, then confidently launch themselves into the clouds. A few minutes later you receive a message from Esquiline station informing you that the first void sprite just swooped through the bay, grabbed one of the asteroid supplements, and vanished into the clouds again.`
+	on complete
+		log "Succesfully introduced a flock of void sprites to Esquiline. Time will tell how they fare."
+		"salary: Remnant" = 2500
+		payment 1000000
+		event "remnant: pantica void sprites"
+		event "remnant: cognizance timer" 30
+		conversation
+			`	Moments later a ping comes in on the comlink, and Prefect Chilia and Plume appear onscreen. "Thank you for your work, Captain. It is unclear what the results of this effort will be, but for now, we wait and see. Hopefully the void sprites settle in," concludes Plume.`
+			`	"In the meantime," continues Chilia. "We are allocating you a daily stipend from our resources to cover some of your ongoing crew, maintenance, and upgrading costs. A larger one-time allotment has already been credited to your account with the shipyard facilities to help cover costs incurred during this mission."`
+
+
+
+event "remnant: pantica void sprites"
+	system "Pantica"
+		add fleet "Void Sprites" 4000
+	planet "Esquiline"
+		government "Remnant"
+		add attributes remnant
+		remove attributes uninhabited
+		description `While Esquiline used to be simply another planet the Remnant harvested valuable resources from, the introduction of void sprites and the activation of their station has turned this world into a hidden research lab: The first human astrobiology station.`
```

Comment:
Changed the conversation slightly. So now it is " The first human astrobiology station studying void sprites." Because even with additional missions, I strongly doubt that any future mission chains could have humans building a cloud-giant space station in Nenia before this point in the Remnant story. (Actually I strongly doubt that it would make sense to ever have it happen, but it should definitely be much later in the game than the Remnant stuff.)